test/integration/test.speech_to_text.js

```diff
@@ -77,14 +77,14 @@ describe('speech_to_text_integration', function() {
                 {
                   "normalized_text": "rain",
                   "start_time": 5.59,
-                  "confidence": 0.986,
+                  "confidence": 0.994,
```

Comment:
I think this is flakeyness at the service level.. because other tests are passing with the old values. I think we are going to have to either loosen up this test or else skip it entirely for now.